ansible/site.yml

```diff
@@ -14,27 +14,95 @@
 
 ---
 # Defines deployment design and assigns role to server groups
+- name: deploy commom
```

Comment:
common is not a standalone role, it's like a public library imported by other rules.

Reply:
common is a pre-deployment operation，so I changed its name to "pre-deployment operation" . Please see if this is feasible.